### PR TITLE
Add `*_MONTH*`, `*_YEAR*` to `lang/time.txt`

### LINKED ISSUE #968: [Request] Add `*_MONTH*`, `*_YEAR*`  to `lang/time.txt`
Add this for all languages here https://github.com/alliedmodders/amxmodx/blob/master/plugins/lang/time.txt#L11

```ini
[ru]
TIME_ELEMENT_MONTH = месяц
TIME_ELEMENT_MONTHS = месяца
TIME_ELEMENT_YEAR = год
TIME_ELEMENT_YEARS = лет

<...>

[en]
TIME_ELEMENT_MONTH = month
TIME_ELEMENT_MONTHS = months
TIME_ELEMENT_YEAR = year
TIME_ELEMENT_YEARS = years
```

### PR DESCRIPTION
**Pull request by wopox1337**
resolves #968

```ini
[en]
TIME_ELEMENT_MONTH = month
TIME_ELEMENT_MONTHS = months
TIME_ELEMENT_YEAR = year
TIME_ELEMENT_YEARS = years
```

A request to those who have looked at this page. Please add translations in other languages (google translate does not look good). Indicate your translations in the comments. Thanks for understanding! 🙂

P.S: I hope this update will be included in AMXX 1.9.0 (not only in AMXX 1.10.0)

### CODE REVIEW

**Comment by regener** [comment]
>[hu]
TIME_ELEMENT_MONTH = hónap
TIME_ELEMENT_MONTHS = hónapok
TIME_ELEMENT_YEAR = év
TIME_ELEMENT_YEARS = évek

**Comment by AdamRichard21st** [comment]
```ini
[bp]
TIME_ELEMENT_MONTH = mês
TIME_ELEMENT_MONTHS = meses
TIME_ELEMENT_YEAR = ano
TIME_ELEMENT_YEARS = anos
```

**Comment by wopox1337** [comment]
hm... Do I need to consider the leap second?  🧐
https://en.wikipedia.org/wiki/Leap_second

**Comment by wopox1337** [comment]
All that remains is to add ALL other translations into other languages. I suppose it is okay if some are missing from the beginning since the basic [EN] will be taken.

Or maybe it makes sense to translate everything via Google Translate (with minor translation errors, of course).

I would like to point out that there is a plural, singular, nominative problem, but that does not apply to this PR. (Maybe someday someone will have time to make a PR fixing this behavior)
```sourcepawn
enum _:
{
    Plural,
    Singular,
    Nominative
};


stock get_numerical_noun_form(iNum)
{
    if (iNum > 10 && ((iNum % 100) / 10) == 1)
        return Plural;                   

    switch (iNum % 10)
    {
        case 1: return Nominative;       
        case 2, 3, 4: return Singular;   
    }

    return Plural;                       
}
```

File: plugins/lang/time.txt [hunk 1]
```diff
@@ -37,6 +45,10 @@ TIME_ELEMENT_DAY = dan
 TIME_ELEMENT_DAYS = dani
 TIME_ELEMENT_WEEK = nedelja
 TIME_ELEMENT_WEEKS = nedelje
+TIME_ELEMENT_MONTH = mesec
+TIME_ELEMENT_MONTHS = meseci
+TIME_ELEMENT_IEAR = godina
```
Comment (WPMGPRoSToTeMa) [inline plugins/lang/time.txt:50]: Typo:
```suggestion
TIME_ELEMENT_YEAR = godina
```

File: plugins/lang/time.txt [hunk 2]
```diff
@@ -37,6 +45,10 @@ TIME_ELEMENT_DAY = dan
...
+TIME_ELEMENT_IEARS = godine
```
Comment (WPMGPRoSToTeMa) [inline plugins/lang/time.txt:51]: The same typo as above:
```suggestion
TIME_ELEMENT_YEARS = godine
```

**Review by WPMGPRoSToTeMa — CHANGES_REQUESTED** [review]

File: plugins/lang/time.txt (hunk 1 above)
Comment (wopox1337) [reply plugins/lang/time.txt]: done https://github.com/alliedmodders/amxmodx/pull/969/commits/3dbfbb6456420cc3f72bdacdf83fe4b896fb4a24

File: plugins/lang/time.txt (hunk 2 above)
Comment (wopox1337) [reply plugins/lang/time.txt]: done https://github.com/alliedmodders/amxmodx/pull/969/commits/3dbfbb6456420cc3f72bdacdf83fe4b896fb4a24

**Comment by Einyux** [comment]
[fr]
TIME_ELEMENT_MONTH = mois
TIME_ELEMENT_MONTHS = mois
TIME_ELEMENT_YEAR = année
TIME_ELEMENT_YEARS = années

And you can update too:
TIME_ELEMENT_PERMANENTLY = à vie (permanent)

**Comment by wopox1337** [comment]
@Arkshine What else is required for this PR to be accepted?

**Comment by FEDERICOMB96** [comment]
```
[es]
TIME_ELEMENT_MONTH = mes
TIME_ELEMENT_MONTHS = meses
TIME_ELEMENT_YEAR = año
TIME_ELEMENT_YEARS = años
```

**Review by Nextra — APPROVED** [review]

**Comment by TibyXD** [comment]
[RO]

TIME_ELEMENT_DAYS = zile
TIME_ELEMENT_WEEK = săptămână
TIME_ELEMENT_WEEKS = săptămâni
TIME_ELEMENT_MONTH = lună
TIME_ELEMENT_MONTHS = luni
TIME_ELEMENT_YEAR = an
TIME_ELEMENT_YEARS = ani
TIME_ELEMENT_PERMANENTLY = permanent
TIME_ELEMENT_AND = și

**Pull request merged**